src/coreclr/tools/aot/ILCompiler.ReadyToRun/JitInterface/CorInfoImpl.ReadyToRun.cs

```diff
@@ -2604,5 +2603,123 @@ private void reportInliningDecision(CORINFO_METHOD_STRUCT_* inlinerHnd, CORINFO_
                 _inlinedMethods.Add(inlinee);
             }
         }
+
+        /// <summary>
```

Comment:
This method (ResolveVirtualStaticMethod) and TryResolveVirtualStaticMethodOnThisType should be part of MetadataVirtualMethodAlgorithm.

Reply:
Moved in 2nd commit, thanks for pointing that out!